user-story-template.md

```diff
@@ -0,0 +1,11 @@
+### Title:
+- Priority:
+- Estimate:
+#### Details:
+- As a <type_of_user>
+- I want to <perform_some_task>
+- so that I can <achieve_some_goal>
+#### Acceptance Criteria
+- Given <some_context>
+- When <some_action_carried_out>
+- Then <set_of_observable_outcomes>
```

Comment:
I think GitHub-flavored Markdown task lists would be a nice touch for the Acceptance Criteria. Instead of lists that begin with `-`, they look like this:

``` md
- [ ] Given <some_context>, when <some_action_carried_out> then <set_of_observable_outcomes>
- [ ] ... ac 2
- [ ] ... ac 3
- [ ] etc.
```

And look like this when rendered:

- [ ] Given <some_context>, when <some_action_carried_out> then <set_of_observable_outcomes>
- [ ] ... ac 2
- [ ] ... ac 3
- [ ] etc.

Reply:
agreed, I didn't even think of those.

Reply:
So, something like this?
`
- [ ] Given <some_context>, when <some_action_carried_out>, Then:
  - [ ] <observable_outcome_1>
  - [ ] <observable_outcome_2>
`

Reply:
Ooh, yeah, that's really nice and would be a great starting point for testing as well!